.github/workflows/ruby.yml

```diff
@@ -0,0 +1,43 @@
+name: Ruby CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        exclude:
+          - ruby-version: 2.3.8
+            gemfile: gemfiles/activesupport_6.gemfile
+          - ruby-version: 2.4.10
+            gemfile: gemfiles/activesupport_6.gemfile
+        gemfile:
+          - gemfiles/activesupport_5.gemfile
+          - gemfiles/activesupport_6.gemfile
+        ruby-version:
+          - 2.3.8
```

Comment:
Just curious, are these the ruby versions we were previously testing against? For context 2.6 is EOL in March this year, so I'm curious.

We could always open an issue to track in the runbook library to drop support for older versions and bump the major version of the gem.

Reply:
@hollabaq86 - thanks for the review. Yep, [these were what we were testing against](https://github.com/braintree/runbook/blob/6f4d226c605c117bc8da88a74d57e68aca4196c4/.travis.yml#L6-L9) before in Travis.

I do think we should work on dropping support for older versions. That's a great callout.